fix(apple): Not taking screenshots for all crashes (#13331)
getsentry/sentry-docs: docs/platforms/apple/common/enriching-events/screenshots/index.mdx

  - apple.watchos
---

> Sentry makes it possible to automatically take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink> when a user experiences an error, an exception or a crash. When a crash occurs on Apple operating systems, there is no guarantee that the Cocoa SDK can still capture a screenshot because, strictly speaking, you're not allowed to call any async unsafe code. Still, it works in most cases, but we can't guarantee screenshots for all types of crashes.

Reviewer: ```suggestion
Sentry makes it possible to automatically take a screenshot and include it as an <PlatformLink to="/enriching-events/attachments/">attachment</PlatformLink> when a user experiences an error, an exception or a crash. When a crash occurs on Apple operating systems, there is no guarantee that the Cocoa SDK can still capture a screenshot since you're technically not allowed to call any async unsafe code. While it still works in most cases, we can't guarantee screenshots for all types of crashes.
```
